osquery/tables/networking/linux/iptables.cpp

```diff
@@ -115,8 +150,12 @@ void genIPTablesRules(const std::string &filter, QueryData &results) {
 
       if (chain_rule->target_offset) {
         r["match"] = "yes";
+        // fill protocol port details
+        parseEntryMatch(chain_rule, r);
       } else {
         r["match"] = "no";
+        r["src_port"] = "";
```

Comment:
By this point we have three different things these values can be for various (error?) cases. If this is intentional, can you document what they all mean and when you might get them? If not, or it's not important, let's make them all the same to make it easier to join the tables later.